sample/src/main/java/com/google/accompanist/sample/navigation/material/BottomSheetNavSample.kt

```diff
@@ -101,11 +101,15 @@ private fun HomeScreen(showSheet: () -> Unit, showFeed: () -> Unit) {
 
 @Composable
 private fun BottomSheet(showFeed: () -> Unit, showAnotherSheet: () -> Unit, arg: String) {
-    Text("Sheet with arg: $arg")
-    Button(onClick = showFeed) {
-        Text("Click me to navigate!")
-    }
-    Button(onClick = showAnotherSheet) {
-        Text("Click me to show another sheet!")
```

Comment:
@jossiwolf These UI components aren't wrapped in a wrapper. That's why they are getting stacked on top of each other.